planner/service/plannerconf.yml

```diff
@@ -5,7 +5,7 @@ server:
   connector:
     type: http
     port: 1234
-discovererURL: undefined
```

Comment:
If I understood well are they default values just for testing on your machine? It reminds me to https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/dashboard/config.yml but instead of hardcoding those values, I have them wired with env variables (which are added in the launch configuration in InteliJ)